import static org.junit.Assert.fail;

import java.sql.Connection;

import javax.sql.DataSource;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.test.context.ContextConfiguration;
import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;

@RunWith(SpringJUnit4ClassRunner.class)
@ContextConfiguration("file:src/main/resources/datasource.xml")
public class DataSourceTest {

	@Autowired
	DataSource dataSource;

	@Test
	public void 데이터소스연결() {
		Connection conn = null;
		try (
				conn = dataSource.getConnection()) {
			System.out.println(conn);
		} catch (Exception e) {
			System.out.println(e.getMessage());
		}
	}
}